lib/extensions/src/lib/services/app-extension.service.ts

```diff
@@ -29,8 +30,17 @@ export class AppExtensionService {
     references$: Observable<ExtensionRef[]>;
     private _references = new BehaviorSubject<ExtensionRef[]>([]);
 
-    constructor(protected extensionService: ExtensionService) {
+    constructor(protected extensionService: ExtensionService, authenticationService: AuthenticationService, appConfigService: AppConfigService, apiService: AlfrescoApiService) {
         this.references$ = this._references.asObservable();
+
+        authenticationService.onLogin.subscribe(() => {
+            const instanceId = appConfigService.get('instanceId', '1234');
```

Comment:
could you please set type for this variable because actually it's any? 
<img width="668" alt="image" src="https://github.com/user-attachments/assets/62f7b297-e926-41ef-b944-ffc259e39f7e" />